fabtests/pytest/efa/test_rdm.py

```diff
@@ -125,36 +133,36 @@ def test_rdm_tagged_peek(cmdline_args):
 # This test is run in serial mode because it takes a lot of memory
 @pytest.mark.serial
 @pytest.mark.functional
-def test_rdm_pingpong_1G(cmdline_args, completion_semantic, fabric):
+def test_rdm_pingpong_1G(cmdline_args, completion_semantic):
```

Comment:
Add TODO to enable test after fixing fi_rdm_pingpong

Reply:
You cannot run pingpong with 1GB with efa direct

Reply:
Oh I thought this was RMA pingpong. Will be nice to have RMA pingpong up to 1G. Can be a follow up.

Reply:
We already have `test_rma_bw_1G`?